Gems/ScriptCanvas/Code/Editor/Components/EditorScriptCanvasComponent.cpp

```diff
@@ -424,14 +424,14 @@ namespace ScriptCanvasEditor
 
     void EditorScriptCanvasComponent::OnAssetReady(const ScriptCanvasMemoryAsset::pointer asset)
     {
-        OnScriptCanvasAssetReady(asset);
+        // OnScriptCanvasAssetReady(asset);
     }
 
     void EditorScriptCanvasComponent::OnAssetSaved(const ScriptCanvasMemoryAsset::pointer asset, bool isSuccessful)
     {
         if (isSuccessful)
         {
-            OnScriptCanvasAssetReady(asset);
+            // OnScriptCanvasAssetReady(asset);
```

Comment:
are these no longer needed? do we need to connect to the asset bus anymore?

Reply:
I don't think they are needed, but I don't want to take them out yet. This file appears to be an accidental submission.